packages/components/modal/src/Modal.tsx

```diff
@@ -73,6 +73,11 @@ export interface ModalProps extends CommonProps {
    */
   allowHeightOverflow?: boolean;
 
+  /**
+   * CSS class to be appended to the overlay element
+   */
+  overlayClassName?: string;
```

Comment:
To align with our prop naming conventions this should go in an `overlayProps` object prop instead, so that you can use the modal like this:

```tsx
<Modal overlayProps={{ className: 'something' }} />
```

Reply:
should it be the `CommonProps` interface?

Reply:
hmm, we can't pass `testId` to overlay element as a prop, in this case we would need to render it ourselves. so should i just declare it like `overlayProps: { className?: string}` or render a custom overlay?

Reply:
Hmm I think let's leave out the `testId` for now and go with just the `className` 🤔